docs/model_cards/geneformer.md

```diff
@@ -104,14 +131,25 @@
 from helical.models.geneformer.model import Geneformer,GeneformerConfig
 import anndata as ad
 
-model_config = GeneformerConfig(batch_size = 10)
-geneformer = Geneformer(model_config = model_config)
-ann_data = ad.read_h5ad("dataset.h5ad")
-dataset = geneformer.process_data(ann_data)
-embeddings = geneformer.get_embeddings(dataset)
+# For Version 1.0
+model_config_v1 = GeneformerConfig(model_name="gf-12L-30M-i2048", batch_size=10)
+geneformer_v1 = Geneformer(model_config=model_config_v1)
+
+#For Version 2.0
+model_config_v2 = GeneformerConfig(model_name="gf-12L-95M-i4096", batch_size=10)
+geneformer_v2 = Geneformer(model_config=model_config_v2)
+
```

Comment:
I think we need more detail here around the continual learning part. I.e. What is different and when to use it